src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala

```diff
@@ -306,7 +313,7 @@ class TezosDatabaseOperationsTest
       implicit val randomSeed = RandomSeed(testReferenceTimestamp.getTime)
 
       //generate data
-      val blocks = generateBlockRows(2, testReferenceTimestamp)
+      val blocks @ (last :: first :: genesis :: Nil) = generateBlockRows(toLevel = 2, startAt = testReferenceTimestamp)
```

Comment:
genesis is kinda "foundation", it's the taken-for-granted root.
Then the regular blocks builds on top of genesis, so first is the first "real" block, and last the latest (actually second)

Please suggest better naming if it might help, keeping in mind that the genesis is "special", and not to be confused with the others, it's a corner case element

Reply:
I agree, naming is a bit confusing, but as there are 3 blocks, why not name just `secondBlock :: firstBlock :: genesis :: Nil` - this could avoid confusion

Reply:
renamed